design.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
+
+const isDev = process.env.NODE_ENV !== 'production'
+
+const icons = ['ab-testing']
+
+if (isDev) icons.push('check')
+
+const iconsRoot = path.join(
+  path.dirname(require.resolve('@mdi/svg/package.json')),
+  './svg'
+)
+
+const config = {
+  icons: {
+    path: iconsRoot,
+    files: icons,
+  },
```

Comment:
I am not sure it's handled correctly. Let's imagine, we have icons in 2 packages or 2 different directories. We want to resolve one list from the first directory and another one from the second directory. Is it handled like that before merging config objects?

By the way, this an extra API I wanted to provide to the user to let them list only filenames without full paths. This is what I was talking about in the previous review.

Reply:
So it's possible to use different icon package in design and different icon package in dashboard but it's not possible to use icons from different packages in the same project unless you provide full paths. It's not ideal however we don't need such a config at the moment.

Reply:
That is why I wanted to have a custom merge function: to merge half-prepared configs and then finish expanding all the parameters. I assume this would fix the issue we have here.

I understand it's not needed at the moment, so I resolve the comment but I will create another issue for this.

Reply:
sure, it's good to have it in backlog